R/performance_reliability.R

```diff
@@ -1,22 +1,64 @@
 #' Random Effects Reliability
 #'
-#' Variability-Over-Uncertainty Ratio (d-vour).
+#' @description These functions provide information about the reliability of
+#' group-level estimates (i.e., random effects) in mixed models. They are useful
+#' to assess whether there the predictors yield consistent group-level variability.
+#' "Group-level" can refer, for instance, to different participants in a study,
+#' and the predictors to the effect of some experimental condition.
 #'
-#' @description TODO: Add description.
+#' The conceptually related functions are implemented, `performance_reliability()`,
+#' based on Rouder & Mehrvarz (2024), and `performance_dvour()` (d-vour), based on the
+#' Variability-Over-Uncertainty Ratio between random effects coefficient variability
+#' and their assocaited uncertainty.
 #'
-#' @param x A model object (or from [`modelbased::estimate_grouplevel()`]).
+#'
+#' @param x A model object.
 #' @param ... Currently not used.
 #'
 #
 #'
-#' @details TODO: Add details.
+#' @details
+#' ## Reliability (Signal-to-Noise Ratio)
+#' `performance_reliability()` estimates the reliability of random effects (intercepts and slopes)
+#' in mixed-effects models using variance decomposition. This method follows the **hierarchical modeling**
+#' framework of Rouder & Mehrvarz (2024), defining reliability as the **signal-to-noise variance ratio**:
+#'
+#' \deqn{\gamma^2 = \frac{\sigma_B^2}{\sigma_B^2 + \sigma_W^2}}
+#'
+#' where:
+#' - \eqn{\sigma_B^2} is the **between-subject variance** (i.e., variability across groups).
+#' - \eqn{\sigma_W^2} is the **within-subject variance** (i.e., trial-level measurement noise).
+#'
+#' This metric quantifies **how much observed variability is due to actual differences between groups**,
+#' rather than measurement error or within-group fluctuations.
+#'
+#' To account for **trial count (\eqn{L})**, reliability is adjusted following:
+#'
+#' \deqn{E(r) = \frac{\gamma^2}{\gamma^2 + 1/L}}
+#'
+#' where \eqn{L} is the number of **observations per random effect level** (note
+#' that Rouder (2024) recommends 2/L to adjust for contrast effects).
+#'
+#'
+#' ## Variability-Over-Uncertainty Ratio (d-vour)
+#' `performance_dvour()` computes an alternative reliability measure based on the **ratio of observed variability
+#' to uncertainty in random effect estimates**. This is defined as:
+#'
+#' \deqn{\text{D-vour} = \frac{\sigma_B^2}{\sigma_B^2 + \mu_{\text{SE}}^2}}
+#'
+#' where:
+#' - \eqn{\sigma_B^2} is the **between-group variability** (computed as the SD of the random effect estimates).
+#' - \eqn{\mu_{\text{SE}}^2} is the **mean squared uncertainty** in random effect estimates (i.e., the average uncertainty).
+#'
+#'
+#' ### Interpretation:
+#' - **D-vour > 0.75**: Strong group-level effects (between-group variance is at least 3× greater than uncertainty).
```

Comment:
The `3×` _might_ cause troubles rendering the PDF (not sure, but _can_ be), so we must remember this when submission to CRAN complains about invalid chars when creating the PDF manual.